<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductManagerForm</class>
 <widget class="QWidget" name="ProductManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>189</width>
         <height>310</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="idLabel">
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="idLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="productNameLabel">
              <property name="text">
               <string>productName</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="productNameLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="priceLabel">
              <property name="text">
               <string>Price</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="priceLlineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="stockLabel">
              <property name="text">
               <string>Stock</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="stockLlineEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="categoryLabel">
              <property name="text">
               <string>Category</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="categoryLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="clearPushButton">
            <property name="text">
             <string>&amp;Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>78</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="addPushButton">
                <property name="text">
                 <string>&amp;Add</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="modifyPushButton">
                <property name="text">
                 <string>&amp;Modify</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>189</width>
         <height>310</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="searchTableView">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QComboBox" name="searchComboBox">
              <item>
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>제품명</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>가격</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>재고량</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>품목</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="searchLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>idLineEdit</tabstop>
  <tabstop>productNameLineEdit</tabstop>
  <tabstop>priceLlineEdit</tabstop>
  <tabstop>stockLlineEdit</tabstop>
  <tabstop>categoryLineEdit</tabstop>
  <tabstop>clearPushButton</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>modifyPushButton</tabstop>
  <tabstop>searchTableView</tabstop>
  <tabstop>searchComboBox</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
